en/config/actuators.md

```diff
@@ -109,31 +146,104 @@ Selecting these requires a reboot before they are applied.
+
+QGC Supports a feature of identifying the motors by sending the motor command to different output pins and letting you click on the motor to map the Motor to each Output ports.
+
+To use this mode:
+1. Setup the Motors Geometry correctly to include all the motors in your vehicle you want to configure
+2. Click the 'Identify & Assign Motors' Button
+3. Each motor will start spinning. Select the motor that spun up in the GUI.
+![](../../assets/config/actuators/identify_motors_in_progress.png)
+   * Click 'Spin Motor Again' if you want to check which motor spun again
+4. After assigning all motors, it will automatically exit and set the Actuator Outputs settings to map correct motors to different outputs.
+
+:::note
+This is only supported for Multirotors currently, as it requires a GUI interface that shows where each motor is located at
+:::
+
+### Actuator setup using Slider
+![Actuator Testing Slider](../../assets/config/actuators/vtol_tiltrotor_sliders_example.png)
+
+> Slider testing example with a VTOL Tiltrotor setup
+
+Sliders can be used to verify the following:
+
+1. Motors spin at the **minimum command**
+2. Motors give **positive thrust** in the expected direction
+3. **Control Surface** moves in the direction as defined in the [Control Surface Convention](#control-surface-direction-conceptual)
+4. **Motor Tilt Servos** move in the direction as defined in the [Tilt Servo Convention](#motor-tilt-servos-conceptual)
+
+:::note
+The Slider has a Safety Feature built in so that even after you toggle the "Enable sliders" switch, if you don't move the slider, it won't send actuator control commands. This is to prevent unwanted action from happening just after enabling the sliders.
+:::
+
+More detailed setup directions are described below.
+
+#### Motor setup & testing
+:::note
+Always remove the propellers from the motor before you do the Actuator testing for Safety!!
+:::
 
-Note the following behaviour:
+1. Pull the Motor slider down, and it will snap in lower end, where it will command **disarmed** value to the motors, verify that it doesn't spin
+   1. For PWM output, adjust the Disarmed value in case the motor spins at the disarmed command
+2. Slowly move the Motor slider up, and it will snap in at the '**minimum** command'. Verify that the motor is spinning at the minimum speed.
+   1. For **PWM output, adjust the minimum output** value in the 'Actuator Outputs' tab on the right such that the motors barely spin
+    ![PWM Minimum Output](../../assets/config/actuators/pwm_minimum_output.png)
+   :::note
+   For DShot output, this is not required
+   :::
+3. Move the slider up even more, and verify that the Motor is spinning in the correct direction and that it would give a positive thrust in the expected direction
+   1. The expected thrust direction can vary by vehicle type, for example in multicopters the thrust should always point upwards
+   2. For Fixed Wings, the thrust should point to forward direction
```

Comment:
Isn't that too basic?

Reply:
What do you mean by basic?

Reply:
Beat means "simple". Even the most ignorant user will know that thrust is forwards on a fixed wing vehicle.